gulpfile.js

```diff
@@ -60,16 +51,41 @@ gulp.task('default', ['clean'], function() {
       path.basename = file_name
       path.extname = ".html"
     }))
+    .pipe(html_replace({
```

Comment:
Does `html_replace` provides additional functionality that `replace` (see line 48) doesn't provide? Otherwise can we just use `replace`?